client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -80,7 +82,11 @@ export default React.createClass( {
 			} );
 		}
 
-		if ( ModalViews.GALLERY !== activeView && selectedItems.length && userCan( 'upload_files', site ) ) {
+		const canDeleteItems = selectedItems.length && ! selectedItems.some( ( item ) => {
```

Comment:
Alternatively, it may be more readable to simply pull in [`_.every`](https://lodash.com/docs#every), which is designed for this use case (i.e. opposite of `some`).

Also, the `selectedItems.length` check is probably unnecessary.